plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtMigrateCommandWrapper.java

```diff
@@ -383,10 +383,10 @@ private String replaceStorage(String xmlDesc, Map<String, MigrateCommand.Migrate
         return getXml(doc);
     }
 
-    private String getPathFromSourceFileDevText(Set<String> paths, String sourceFileDevText) {
-        if (paths != null && sourceFileDevText != null) {
+    private String getPathFromSourceText(Set<String> paths, String sourceText) {
+        if (paths != null && sourceText != null) {
```

Comment:
What do you think about `if (paths != null && !StringUtils.isBlank(sourceText))`?